src/cascadia/TerminalApp/HighlightedTextControl.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<UserControl
+    x:Class="TerminalApp.HighlightedTextControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:TerminalApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">

```

Comment:
Hmm. This one gets me -- should we force the color here or make it transparent? Should we actually template this control and offer a Background binding?

Reply:
(Easiest is to make it transparent)

Reply:
Absolutely! Thanks!